worlds/stardew_valley/options.py

```diff
@@ -238,9 +259,23 @@ class BuildingProgression(Choice):
     option_progressive_early_shipping_bin = 2
 
 
+class FestivalLocations(Choice):
+    """Locations for attending and participating in festivals
+    With Disabled, you do not need to attend festivals
```

Comment:
May be worth noting that you can still be required to go to the Night Market in order to fish. This could happen with Fishsanity or to get Pearls from a Blobfish.

Reply:
I think I'm fine with this